src/index.html

```diff
@@ -5,9 +5,51 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/style.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product">
+      <div class="product__card" data-qa="card">
+        <img
+          class="product__image"
+          src="images/imac.jpeg"
+          alt="iMac"
+        >
+
+        <p class="product__name">
+          APPLE A1419 iMac 27" Retina
+          <br>
+          5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="product__code">Product code: 195434</p>
+
+        <div class="product__rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="product__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="product__price">
+          Price:
+
+          <h1 class="product__price-count">$2,199</h1>
```

Comment:
`h1` tag is used for a page title
```suggestion
          <p class="product__price-count">$2,199</p>
```